.github/workflows/build.yaml

```diff
@@ -201,6 +206,154 @@ jobs:
+      - name: Download leap builddir
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{matrix.platform}}-build
+      - name: Extract leap build
+        run: |
+          zstdcat build.tar.zst | tar x
+      - name: leap dev-install
+        run: |
+          cmake --install build
+          cmake --install build --component dev
+      - name: Delete leap artifacts
+        run: |
+          rm -r *
+      - name: Download cdt
+        uses: AntelopeIO/asset-artifact-download-action@v2
+        with:
+          owner: AntelopeIO
+          repo: cdt
+          file: 'cdt_.*amd64.deb'
+          target: main
+          artifact-name: cdt_ubuntu_package_amd64
+          token: ${{github.token}}
+      - name: Install cdt Packages
+        run: |
+          apt install -y ./*.deb
+          rm ./*.deb
+      - name: checkout reference-contracts
+        uses: actions/checkout@v3
+        with:
+          repository: AntelopeIO/reference-contracts
+          path: reference-contracts
+      - name: Build & Test reference-contracts
+        run: |
+          cmake -S reference-contracts -B reference-contracts/build -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=On
+          cmake --build reference-contracts/build -- -j $(nproc)
+          cd reference-contracts/build/tests
+          ctest --output-on-failure -j $(nproc)
+
+  libtester-build-tree-test:
+    name: libtester build tree test
+    needs: [d, Build]
+    if: always() && needs.Build.result == 'success'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu20, ubuntu22]
+    runs-on:  ["self-hosted", "enf-x86-lowtier"] # not sure if this warrants a better machine, but start here
+    container: ${{fromJSON(needs.d.outputs.p)[matrix.platform].image}}
+    steps:
+      - name: Clone leap
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Download leap builddir
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{matrix.platform}}-build
+      - name: checkout reference-contracts
+        uses: actions/checkout@v3
+        with:
+          repository: AntelopeIO/reference-contracts
+          path: reference-contracts
+      - name: Extract leap build
+        run: |
+          zstdcat build.tar.zst | tar x
+          echo "leap_DIR=$PWD/build/lib/cmake/leap" >> "$GITHUB_ENV"
+          echo "LEAP_BUILD_DIR=$PWD/build" >> "$GITHUB_ENV"
+      - name: Download cdt
+        uses: AntelopeIO/asset-artifact-download-action@v2
+        with:
+          owner: AntelopeIO
+          repo: cdt
+          file: 'cdt_.*amd64.deb'
+          target: main
+          artifact-name: cdt_ubuntu_package_amd64
+          token: ${{github.token}}
+      - name: Install cdt Packages
+        run: |
+          apt install -y ./*.deb
+          rm ./*.deb
+      - name: Build & Test reference-contracts
+        run: |
+          export leap_DIR="${{ env.leap_DIR }}"
```

Comment:
Isn't this just setting the value to what it already has, from here
https://github.com/AntelopeIO/leap/blob/06373d2cfc59bdefb362e9af9259713c963306ac/.github/workflows/build.yaml#L334-L335
my guess is we only need that one single line in its current location
https://github.com/AntelopeIO/leap/blob/06373d2cfc59bdefb362e9af9259713c963306ac/.github/workflows/build.yaml#L334